docs/source/api_reference/polygon.rst

```diff
@@ -19,6 +22,86 @@ Example
 
     m
 
+.. raw:: html
+
+  <script type="application/vnd.jupyter.widget-view+json">
+  {
+    "model_id": "c7eeeba10da14d6d952eb47205ba4e79",
+    "version_major": 2,
+    "version_minor": 0
+  }
+  </script>
+  <div style ="height:30px;"> </div>
+
+
+
+Example Polygon with hole
+-------------------------
+
+.. code::
+
+  from ipyleaflet import Map, Polygon
+
+  Hpolygon = Polygon(
```

Comment:
It's unusual to use a capital letter for variable names, we generally use a capital letter for class names. You can just name it `polygon`